Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockObjectFactory.cs

```diff
@@ -19,46 +19,51 @@
  */
 
 using FluentAssertions;
+using Moq;
+using SonarScanner.MSBuild.Common;
+using SonarScanner.MSBuild.Common.TFS;
+using SonarScanner.MSBuild.PreProcessor.Roslyn.Model;
+using TestUtilities;
 
 namespace SonarScanner.MSBuild.PreProcessor.Test
```

Comment:
```suggestion
namespace SonarScanner.MSBuild.PreProcessor.Test.Infrastructure
```

Reply:
None of the files in that directory has extra namespace. I'd leave it like that